### ISSUE TITLE
Update civi with all staff rolodexes before Global Summit

### ISSUE THREAD

**Issue by rebeccalendl** [creativecommons/creativecommons.org#122]
Include all staff contacts (not just address book contacts), widest cast net of people who have engaged with CC over the years
Include all former staff contacts (if those address books are available) 
Import to civi
Merge with existing records, clean up dupes, associate with existing Organizations if any, associate with appropriate staff member
Send welcome note to email list with opt out (not opt in), share reasoning "because of your interest/engagement in CC over the years, we wanted to keep you up to date on all things CC..." (@rebeccalendl can draft)
Add to email list

**Comment by mattl** [comment]
Can you send a reminder tomorrow? So far we have files from Tim.

**Comment by rebeccalendl** [comment]
Following up on this import, please prepare email test in civi by next Friday AM for review on Trilogy call "thanks for being part of our community / welcome to our email list." Email will go to 1) CC staff contacts who are not yet in civi, 2) Civi contacts who are not on our email list, but who have not actively opted out. Email will send Friday following the call, or first thing Monday if needed.

**Comment by rebeccalendl** [comment]
@mattl Hope you are feeling better. Not sure what your schedule is like this week - possible to get the import complete and the welcome email out by Friday?

**Comment by mattl** [comment]
Only one person has sent me anything, so no.

**Comment by rebeccalendl** [comment]
I thought I had heard confirmations from Sarah, Jane, Tim, and myself. Is that not right? 

And we also can send to all emails in civi who are not signed up but have not opted out, yes?

**Comment by mattl** [comment]
I've only seen one from Tim. Can you ask people to double check? They
should be emailing a CSV file to webmaster@creativecommons.org

**Comment by rebeccalendl** [comment]
I just fwded you mine and sarahs which were sent to webmaster. I'll check with the other staff.

**Comment by rebeccalendl** [comment]
What's the status on this?

**Comment by mattl** [comment]
Can you export your contacts again and resend? Doing a couple test runs with Sarah and Tim's and your export is missing a large number of fields.

**Comment by mattl** [comment]
Got approximately half of tvol's imported, a ton of dupes are holding me up now.

**Comment by rebeccalendl** [comment]
I spent a lot of time cleaning up names and adding info so not easy to do a second time. Is it essential or can it be imported as is? If essential, you'll want to please note to full staff by replying to my staff reminder email any specifications you have for their exports before they send you their data (especially for staff with long contact lists who are not likely to go through the activity twice).

**Comment by mattl** [comment]
You've removed a ton of the columns which are present in the export. I'll
just wait until we're done to do yours, hopefully everyone else continues
to send a Google export format.

---
Matt Lee
Creative Commons
Austin, TX, USA
3pm-11pm UTC

**Comment by rebeccalendl** [comment]
please let them know what you need. it will be a challenge to get them all to do it once, and likely not possible to ask them to do it again.

**Comment by mattl** [comment]
Done.

**Comment by rebeccalendl** [comment]
Again checking on status? Waiting on test email to current staff contacts and civi people who have not opted out.

**Comment by mattl** [comment]
Have all of tvol imported. So many dupes are causing a bunch of problems on
the import. When can you get those fixed?

**Comment by rebeccalendl** [comment]
What does "can you get those fixed mean"? I don't have any outstanding instructions that I am aware of.

Also, any reason why we couldn't still send the email out as it would remove any duplicate sends?

**Comment by mattl** [comment]
I thought you and Mari were working on fixing up the dupes?

The problem isn't sending the email, but importing them into Civi. A lot of
people in Tim's contacts were already in Civi with dozens of records. Those
duplicates have to be merged so we can successfully import the contacts and
get them in a place to be emailed.

**Comment by rebeccalendl** [comment]
Are you referring to this? https://creativecommons.freshdesk.com/helpdesk/tickets/15756 

I didn't hear back from you on it yet. The dupe clean up project is massive and ongoing, I was not made aware that this email send was dependent on that. As I've said a bunch of times, the goal of the email send was to synch it with the summit, would be impossible to clean up all of civi before that.

**Comment by mattl** [comment]
I didn't see your reply in Freshdesk until now. You'd need to assign the
ticket to me for me to see it.

The email send isn't dependent on the duplicates, but it does make it a
slow process.

Currently, I have to:

* Import contacts
* Look at error records (missing email addresses, etc)
* About half of them come back as duplicates

When I have the list of duplicates, I have to import them into a temporary
group of their own, run the dedupe on them which takes a while. Once
they're fixed up, then I need to go through the group and manually find
people we can't email because of anti-spam laws in Canada, etc.

It wasn't possible to get things done in time for the summit, as I was out
for several days.

**Comment by rebeccalendl** [comment]
I sent the freshdesk ticket directly to your email address. Wouldn't that mean that you see it even if it wasn't reassigned? 

If you are changing the dependencies on a project, you need to make me aware of that. I did not have it on my task list to do the clean up before the summit. In fact, that would have not been possible - it will likely be completed no sooner than year-end.

I know you were out the entire week before the summit. Were you also out all of last week? I'd like to move on this asap so that we can at least synch it with the summit close out. Otherwise as I've shared before, the first thing our new contacts/community would be hit with will be year end fundraising. 

So is this project not possible now without cleaning up the entire database first? If that is the case, that would be a pretty disappointing since we have been talking about the project for many months now.

**Comment by mattl** [comment]
See my previous comment, this project is certainly possible without
cleaning up the database.

**Comment by rebeccalendl** [comment]
Great. Let me know your timeframe please. And let me know if I can help or if am holding you up at all for any reason. I'd like to review email and recipient list before anything goes out.

**Comment by mattl** [comment]
I should have all the contacts I have imported into their various groups
today. I will run some checks against them to remove anyone with a Canadian
email address, and then emails from staff members to their contacts can be
sent.

I don't think we have any copy for that, so if you want to write something,
that'd be useful. I'm happy to write a few short sentences too, based on
the Kickstarter/Newsletter email.

---
Matt Lee
Creative Commons
Austin, TX, USA
3pm-11pm UTC

**Comment by rebeccalendl** [comment]
Copy is here
https://docs.google.com/document/d/1_b-MMgwpj5Nu2JNpkja_cHCQppiRbJgbvpjF2PIXRDE/edit

Please also plan to send to currently in civi who have not opted out.

**Comment by mattl** [comment]
We should probably tweak it a bit now, as it talks about tuning into the
summit.

Also, we only want one link, so I guess that will be to the summit website?

---
Matt Lee
Creative Commons
Austin, TX, USA
3pm-11pm UTC

**Comment by rebeccalendl** [comment]
I'll finesse copy. You work on recipient list. Can you also confirm my few previous notes about currently in civi not opted out since I think that will be the bulk of our recipients, yes?

**Comment by mattl** [comment]
Are you referring to people outside of the staff contacts, but who are
not-opted out of receiving emails from us but who are not on our
newsletter?

**Comment by rebeccalendl** [comment]
correct

**Comment by mattl** [comment]
Okay, I'll open another issue about that. I did the work a few weeks ago to
check all the email addresses we had, and we have a bunch of them marked as
"do not email" now.

**Comment by mattl** [comment]
All contacts I have now are imported, I will be checking them for
things we need to remove such as Canadian addresses.

**Comment by rebeccalendl** [comment]
Thanks for the update. Copy is final and ready when you are.

**Comment by mattl** [comment]
Added some notes to the copy, we need a few more tweaks still.

**Comment by mattl** [comment]
Just waiting on Ryan now, who we may want to treat differently anyway.
